src/main/drivers/bus_spi.c

```diff
@@ -119,6 +119,9 @@ void initSpi1(void)
     // MISO as input
     gpio.pin = SPI1_MISO_PIN;
     gpio.mode = Mode_IN_FLOATING;
+#ifdef USE_NRF24_SPI1
```

Comment:
... and while I'm at it, thank you very much @martinbudden for cleaning up and merging my code, and @ledvinap for code review.